### ISSUE TITLE
Add DECONV_PARAM table to Irene-produced files

### ISSUE THREAD

**Issue by gonzaponte** [next-exp/IC#191]
We need to add this table to keep track of the params used.

**Comment by gonzaponte** [comment]
The current structure is:
````
class DECONV_PARAM(tb.IsDescription):
    N_BASELINE            = tb.Int32Col(pos=0)
    THR_TRIGGER           = tb.Int16Col(pos=1)
    ACUM_DISCHARGE_LENGTH = tb.Int16Col(pos=2)
    ACUM_TAU              = tb.Int16Col(pos=3)
    ACUM_COMPRESS         = tb.Float32Col(pos=4)
````
 but 
````
    ACUM_DISCHARGE_LENGTH = tb.Int16Col(pos=2)
    ACUM_TAU              = tb.Int16Col(pos=3)
    ACUM_COMPRESS         = tb.Float32Col(pos=4)
````
 do not appear anywhere. @jjgomezcadenas, please comment.

**Comment by jjgomezcadenas** [comment]
Of these 3 parameters only one is still used 
> ACUM_DISCHARGE_LENGTH = tb.Int16Col(pos=2)

The parameter needs to be passed to a DeconvolutionCity. The other two can be eliminated from the table.

**Comment by jmbenlloch** [comment]
Closed by #192.
